features/step_definitions/entrypoint_steps.js

```diff
@@ -1,8 +1,9 @@
 module.exports = function () {
-  this.When(/^I access the tool with my account identifier$/, function (callback) {
-    this.visit(`/?id=${this.scenarioData.accountIdentifier}`, callback);
+  this.When(/^I access the tool with my account identifier$/, function () {
+    return this.dashboardPage.visit({ id: this.scenarioData.accountIdentifier });
```

Comment:
The dashboard page isn't the application root, I think these should just be `this.browser.visit`